guides/advanced_guides/asynchronous_updates.md

```diff
@@ -9,6 +9,7 @@ Some operations are put in a queue and will be executed in turn (asynchronously)
 - When making a write request (_create/update/delete_) against the search engine, it stores the operation received in a queue and returns an `updateId`. With this id, the operation update is trackable.
 - Each update received is treated following the order it has been received.
 - You can get the update status on the [`/updates`](/references/updates.md) route.
+- Once an update has been processed, its user content is removed. The only data that is kept includes the number of documents that were processed and the `updateId`.
```

Comment:
What do you mean by `user` content?

Reply:
The content related to the user's update but your question means I should rephrase it and my explanations weren't clear enough! Let me know if you have any suggestion.

Reply:
I bugged on `user content` too 😉 But sorry I have no suggestion to improve that...
(I don't know if saying "I bugged" is a real word, sorry 😇)

Reply:
I like "bugged". ^^
I post you 2 screenshots. I understood my explanation was totally wrong. They are not really deleted but kept and marked as processed.

![before](https://user-images.githubusercontent.com/41596925/81573193-6ebcad80-93a4-11ea-9845-8bef7874c926.png)

After:
![after](https://user-images.githubusercontent.com/41596925/81573196-70867100-93a4-11ea-964e-fe4937c76a39.png)

Reply:
I think you did not bug but maybe this example does not showcase the problem. Maybe @Kerollmops could enlighten us.

Reply:
The data that was sent by the user, the update content, is kept until it is processed by the database (it is asynchronous). Once processed the only information that is kept by the database is, for example, the number of documents that were processed in this update.